library/jdbc/forage-jdbc-common/pom.xml

```diff
@@ -23,6 +23,10 @@
             <artifactId>forage-core-jdbc</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
It seems like that this is only needed for `org.apache.camel.dsl.jbang.core.common.RuntimeType;`

Since, the camel-jbang-plugin-forage already import this dependency, is it really needed in other modules? can we get rid of it somehow in jdbc?

Reply:
I can definitely use String instead, I thought that having the correct enum would be nicer, or I can move the inteface, I'll check the second option (in that case the enum would stay, but dependency would be gone)

Reply:
fixed